lib/12T/addf1/gf180mcu_osu_sc_12T_addf_1.lef

```diff
@@ -0,0 +1,172 @@
+# Copyright 2022 Google LLC
```

Comment:
`12T` is not a valid name for a standard cell library according to https://bit.ly/open-source-pdks-naming

I would suggest `osu12t` or maybe `osu1v8t12`.